test/integration/crl_test.go

```diff
@@ -1,3 +1,87 @@
 //go:build integration
 
 package integration
+
+import (
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/letsencrypt/boulder/core"
+	"github.com/letsencrypt/boulder/test"
+)
+
+// runUpdater executes the crl-updater binary with the -runOnce flag, and
+// returns when it completes.
+func runUpdater(t *testing.T) {
+	t.Helper()
+
+	config_dir := os.Getenv("BOULDER_CONFIG_DIR")
+	if config_dir == "" {
+		t.Fatal("couldn't determine configuration directory")
+	}
+
+	bin_path, err := filepath.Abs("bin/crl-updater")
```

Comment:
Nit, should be camelCase
```suggestion
	binPath, err := filepath.Abs("bin/crl-updater")
```